.travis/s2n_travis_build.sh

```diff
@@ -29,7 +29,11 @@ fi
 
 # Use prlimit to set the memlock limit to unlimited for linux. OSX is unlimited by default
 if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-    sudo -E "$PRLIMIT_INSTALL_DIR"/bin/prlimit --pid "$$" --memlock=unlimited:unlimited;
+    if [[ -f "$PRLIMIT_INSTALL_DIR"/bin/prlimit ]]; then
+        sudo -E "$PRLIMIT_INSTALL_DIR"/bin/prlimit --pid "$$" --memlock=unlimited:unlimited;
+    else
+        sudo -E /usr/bin/prlimit --pid "$$" --memlock=unlimited:unlimited;
```

Comment:
Should there be a check that this file exists also?

Reply:
I checked in my fresh ubuntu /usr/bin/prlimit is there, as it is just a failover option, we might not need to check it?